gflownet/envs/__init__.py

```diff
@@ -1 +1 @@
-__all__ = ["base", "grid", "aptamers"]
+__all__ = ["base", "grid", "aptamers", "alaninedipeptide"]
```

Comment:
Just out of curiosity, could you explain what this thing is doing?

Reply:
It specifies what will be imported from the module with `from ... import *`. Without it some import errors were raised.